### PR TITLE
Prettier formatting

### PR DESCRIPTION
**Pull request by irace-stripe**
Added [Prettier](https://prettier.io/) formatting.

### CODE REVIEW

**Comment by rattrayalex-stripe** [comment]
cc @paulasjes-stripe @rob-stripe

File: lib/StripeMethod.basic.js [hunk 1]
```diff
@@ -54,39 +53,60 @@ module.exports = {
...
+            metadata[key] = value;
+            sendMetadata(metadata, auth);
+          } else {
+            // Set entire metadata object after resetting it:
+            this._request(
+              'POST',
+              null,
+              path,
+              {
+                metadata: null,
```
Comment (rattrayalex-stripe) [inline lib/StripeMethod.basic.js]: minor, would you mind removing the newline before `metadata` here such that the output is `{metadata: null}`? this is the ~one place that prettier respects original formatting, annoyingly

Comment (rattrayalex-stripe) [reply lib/StripeMethod.basic.js]: (same below)

File: README.md [hunk 2]
```diff
@@ -88,30 +79,53 @@ callback:
...
+
+```js
+var stripe = require('stripe')('sk_test_...');
+
+stripe.customers.create(
+  {
+    email: 'customer@example.com',
+  },
+  function(err, customer) {
+    err; // null if no error occurred
```
Comment (paulasjes-stripe) [inline README.md]: Nit: Can we do something with these values? It looks a little odd by themselves. Perhaps something like:

```
if (err) {
  // Deal with an error
}

// Do something with created customer object
console.log(customer.id) 
```

**Comment by irace-stripe** [comment]
@rattrayalex-stripe @paulasjes-stripe Feedback addressed. Since this was already approved and my changes were trivial, I’m going to merge this.

**Comment by irace-stripe** [comment]
LGTM

**Pull request merged**